src/OrchardCore.Modules/OrchardCore.ContentFields/Views/DateField-Custom.Option.cshtml

```diff
@@ -0,0 +1,4 @@
+@{
+    string currentEditor = Model.Editor;
+}
+<option value="Custom" selected="@(currentEditor == "Custom")">@T["Custom"]</option>
```

Comment:
The name is unfortunate. It should be called `Localized` or `Regional`, something with some meaning.